### PR TITLE
Payment Flow: ui colours and shapes

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

File: public/assets-new-ui/styles.css [hunk 1]
```diff
@@ -256,3 +261,225 @@ header[role="banner"] {
...
+}
+        
+.form__field label {
+  font-weight: bold;
+  line-height: 1;
+  margin-bottom: 4px;
+}
+
+.form__input {
+  -webkit-appearance: none;
```
Comment (Ap0c) [inline public/assets-new-ui/styles.css]: **Comment for refactor**: for prefixes we have [autoprefixer](https://github.com/guardian/support-frontend/blob/master/webpack.common.js#L100) as part of the `postcss` build.

File: app/views/newPaymentFlow.scala.html [hunk 2]
```diff
@@ -5,10 +5,10 @@
 <div class="content">
 
   <h1>Help us deliver the independent journalism the world needs</h1>
-  <p>Make a monthly commitment to support The Guardian long term or a one-off contribution as and when you feel like it. Choose the option that suits you best.</p>
+  <p class="blurb">Make a monthly commitment to support The Guardian long term or a one-off contribution as and when you feel like it. Choose the option that suits you best.</p>
 
   <form action="#" method="post" class="form contribution-form">
-    <fieldset class="form__radio-group form__radio-group--tabs">
+    <fieldset class="contribution-form__type form__radio-group form__radio-group--tabs">
```
[the remark this answers is not in the document]
Comment (regiskuckaertz) [reply app/views/newPaymentFlow.scala.html]: Makes sense! Will update right now.

File: public/assets-new-ui/styles.css [hunk 3]
```diff
@@ -256,3 +261,225 @@ header[role="banner"] {
...
+  display: none;
+}
+
+.form__radio-group ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.form__radio-group ul input {
```
Comment (joelochlann) [inline public/assets-new-ui/styles.css]: there are some CSS guidelines here: https://github.com/guardian/support-frontend/wiki/CSS

as a general rule, as per BEM methodology of avoiding having to think about specificity rules, we would try and keep things as flat as possible and just use class selectors

but this approach might be a temporary waypoint in your iterative process - if so, it's all good!

File: public/assets-new-ui/styles.css [hunk 4]
```diff
@@ -31,6 +32,10 @@ footer[role="contentinfo"] {
   padding: 0;
 }
 
+.support li {
```
[the remark this answers is not in the document]
Comment (regiskuckaertz) [reply public/assets-new-ui/styles.css]: Fair enough. Will do here too.

File: public/assets-new-ui/styles.css (hunk 3 above)
Comment (joelochlann) [reply public/assets-new-ui/styles.css]: sorry - me and @Ap0c literally got to this at the same millisecond

File: public/assets-new-ui/styles.css [hunk 5]
```diff
@@ -256,3 +261,225 @@ header[role="banner"] {
...
+  line-height: 1.4;
+  margin: 3px 0 9px;
+}
+
+footer[role="contentinfo"] {
+  font-family: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
+}
+
+.license small {
+  font-size: 1em;
```
[the remark this answers is not in the document]
Comment (regiskuckaertz) [reply public/assets-new-ui/styles.css]: Gotcha. In this case I want the font to be the same as the parent's, but I can use `inherit` for that.

File: public/assets-new-ui/styles.css (hunk 3 above)
Comment (regiskuckaertz) [reply public/assets-new-ui/styles.css]: Yeah it was, but now that I've sort of finished the big CSS work, I will add class names and move to BEM modifiers 👍

**Comment by regiskuckaertz** [comment]
@joelochlann @Ap0c thanks, have amended with the things I can do now. Will apply the SASS stuff as I think about moving into React-land.

**Pull request merged**